src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -519,6 +571,8 @@ impl<'tcx> Tree {
 
     /// Map the per-node and per-location `LocationState::perform_access`
     /// to each location of `access_range`, on every tag of the allocation.
+    /// If `access_range` is `None`, the access will be applied to
+    /// all initialized locations of the allocation.
```

Comment:
This must definitely mention that the `None` access is visible to children only!